src/job.c

```diff
@@ -31,51 +31,9 @@ this program.  If not, see <http://www.gnu.org/licenses/>.  */
 #include "os.h"
 
 /* Default shell to use.  */
-#ifdef WINDOWS32
-# ifdef HAVE_STRINGS_H
-#  include <strings.h>	/* for strcasecmp, strncasecmp */
-# endif
-# include <windows.h>
-
-const char *default_shell = "sh.exe";
-int no_default_sh_exe = 1;
-int batch_mode_shell = 1;
-HANDLE main_thread;
-
-#elif defined (__MSDOS__)
-
-/* The default shell is a pointer so we can change it if Makefile
-   says so.  It is without an explicit path so we get a chance
-   to search the $PATH for it (since MSDOS doesn't have standard
-   directories we could trust).  */
-const char *default_shell = "command.com";
```

Comment:
I think using command.com as the default shell is pretty weird. Who writes a GNU Makefile assuming that the underlying commands run are being don in the BAT language? 

Such a Makefile could only ever get run on Windows systetms. 

As I've said before, not only is this code bulky and cluttered, it might be of marginal use.  And probably in this day and age the assumptions may even need to be rethought in light of where we are today.

Reply:
Certainly not very clever, I agree but it's only a default setting.

Reply:
But a default that is just about always wrong, is a hinderance rather than a help. 

I thnk you'd be hard pressed to find _any_ software that uses any GNU make with bat or powershell commands. 

Perhaps 35 years ago someone was hoping that GNU Make would be the premium build tool on MS/DOS and MS/Windows.  (Wishful thinking even back then). 

In reality, the only time I've ever seen Make used on those OS's was to run some sort of Unix-y build system.